core/trino-main/src/main/java/io/trino/sql/analyzer/StatementAnalyzer.java

```diff
@@ -728,10 +729,13 @@ protected Scope visitAnalyze(Analyze node, Optional<Scope> scope)
             validateProperties(node.getProperties(), scope);
             CatalogName catalogName = getRequiredCatalogHandle(metadata, session, node, tableName.getCatalogName());
 
+            if (hasDefaultProperty(node.getProperties())) {
```

Comment:
ditto: move property support to `getProperties`

Reply:
Do we want to allow a user to set a property to `DEFAULT` in `ANALYZE`?